README.md

```diff
@@ -1,57 +1,52 @@
 
 ## Getting Started
 
 ### Prerequisites
 
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
-
-### Installation
-
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
+- An [Azure account](https://docs.microsoft.com/dotnet/azure/create-azure-account)
+- [Java Development Kit (JDK)](https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable) with version 8 or above
+- [Maven](https://maven.apache.org/)
+- [VS Code](https://code.visualstudio.com/) or [IntelliJ IDEA](https://www.jetbrains.com/idea/) 
 
 ### Quickstart
-(Add steps to get up and running quickly)
-
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
-
 
-## Demo
-
-A demo app is included to show how to use the project.
-
-To run the demo, follow these steps:
-
-(Add steps to start up the demo)
-
-1.
-2.
-3.
+This repository contains three subdirectories.
+- *starter app* - A starter version of the application without any Azure code.  Intended to be used when following the Quickstart at [https://docs.microsoft.com/azure/cosmos-db/table/create-table-java](https://docs.microsoft.com/azure/cosmos-db/table/create-table-java).
+- *completed app* - A completed version of the app for those who want to see the finished code or debug through a fully working application.
+- *scripts* - Azure CLI and Azure PowerShell scripts to create the Azure resources needed to run this sample.
+
+The fastest way to get started is to run the completed application.  For the application to work, you still need provision the appropriate Azure resources and use the appropriate connection string in the sample application.
+
+1. Clone or download the sample application from https://github.com/Azure-Samples/msdocs-azure-data-tables-sdk-java.git
+2. Use either the Azure CLI or Azure PowerShell script in the scripts directory to create the appropriate Azure resources.  If you do not have Azure CLI or Azure PowerShell installed, you may run either of these scripts in the [Azure Cloud Shell](https://shell.azure.com).
+3. Each script will return the appropriate connection string for the sample application to use. Switch to folder `2-completed-app`.Then, add the value of the `connection string` in `pom.xml` file.
+```
+	<profiles>
+		<profile>
+			<id>local</id>
+			<properties>
+				<azure.tables.connection.string>
+					<![CDATA[YOUR-TABLE-SERVICE-CONNECTION-STRING]]>
+				</azure.tables.connection.string>
+				<azure.tables.tableName>WeatherData</azure.tables.tableName>
+			</properties>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+		</profile>
+	</profiles>
```

Comment:
Using tabs might make this render strangely in some applications.
```suggestion
<profiles>
    <profile>
        <id>local</id>
	<properties>
	    <azure.tables.connection.string>
	        <![CDATA[YOUR-TABLE-SERVICE-CONNECTION-STRING]]>
	    </azure.tables.connection.string>
            <azure.tables.tableName>WeatherData</azure.tables.tableName>
        </properties>
        <activation>
	    <activeByDefault>true</activeByDefault>
	</activation>
    </profile>
</profiles>
```

Reply:
According to your comments, it has been fixed in the new version.